### ISSUE TITLE
Code refactoring - Query Suggestion Service

### ISSUE THREAD

**Issue by PedroG1515** [arquivo/pwa-technologies#1383]
Code refactoring 

The code from query suggestion needs to go inside webapp.

Repo: [PwaSpellchecker](https://github.com/arquivo/PwaSpellchecker)

**Comment by igobranco** [comment]
IMO we should change when we create new value.

**Comment by igobranco** [comment]
I don't agree to merge the code. We should improve the existing Java application or replica it with a new Python implementation. But never ever have a single application that to too much like dealing with the Arquivo.pt user interface and do the query suggestion/spell checking.

**Comment by igobranco** [comment]
replaced by #1467
